tests/queries/0_stateless/03164_parallel_replicas_range_filter_min_max.sql

```diff
@@ -0,0 +1,57 @@
+DROP TABLE IF EXISTS range_filter_custom_range_test;
+
+CREATE TABLE range_filter_custom_range_test (k Int64) ENGINE=MergeTree ORDER BY k;
+
+INSERT INTO range_filter_custom_range_test SELECT number + 5 from numbers(10);
+
+SELECT count() FROM 
+(SELECT *
+FROM cluster(test_cluster_one_shard_three_replicas_localhost, currentDatabase(), range_filter_custom_range_test)
+SETTINGS prefer_localhost_replica = 0, max_parallel_replicas = 3, distributed_group_by_no_merge=0,
+parallel_replicas_custom_key = 'k', parallel_replicas_custom_key_filter_type = 'range', 
+parallel_replicas_custom_key_range_lower = 5, parallel_replicas_custom_key_range_upper=15);
```

Comment:
```suggestion
parallel_replicas_custom_key_range_lower = 5, parallel_replicas_custom_key_range_upper = 15);
```

fix formatting in all tests